ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -527,6 +532,16 @@ public static void get_theoretical_proteoforms(string current_directory)
             }
         }
 
+        //Generate lookup table for ptm sets based on rounded mass of eligible PTMs -- used in forming ET relations
+        public static void make_ptmset_dictionary()
+        {
+            foreach (PtmSet set in all_possible_ptmsets.Where(s => s.ptm_combination.Count == 1 || !s.ptm_combination.Select(ptm => ptm.modification).Any(m => m.monoisotopicMass == 0)))
+            {
+                if (all_possible_ptmset_dictionary.ContainsKey(Convert.ToInt32(set.mass))) all_possible_ptmset_dictionary[Convert.ToInt32(set.mass)].Add(set);
```

Comment:
Since we're not forming peaks with anything outside no man's land right now so that might need to change in the future if we look at glycosylations I suppose